src/frontend/src/lib/components/dapps/DappsExplorerSignedIn.svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import DappCard from '$lib/components/dapps/DappCard.svelte';
+	import DappModal from '$lib/components/dapps/DappModal.svelte';
+	import DappPromoBanner from '$lib/components/dapps/DappPromoBanner.svelte';
+	import Button from '$lib/components/ui/Button.svelte';
+	import { modalDAppDetails } from '$lib/derived/modal.derived';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+	import {
+		type DappDescription,
+		dAppDescriptions,
+		type FeaturedDappDescription
+	} from '$lib/types/dappDescription';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	let selectedTag: string | null = null;
```

Comment:
nitpick: we actually generally used  "something or `undefined`". `null` works for sure just fine but is less consistent